causing/graph.py

```diff
@@ -228,6 +227,11 @@ def specific_color_str(wei: float, line_colored: bool) -> str:
     )
     dot_str = "digraph { \n" + form + x_dot + y_dot + "        }"
 
+    industry = str(dir_path).split("/")[1]
+
+    if industry.startswith("de_"):
+        dot_str = dot_str.replace(".", ",")
```

Comment:
* There's no concept of an industry in Causing. Causing is generic and should not care about industries, companies, countries, etc. Accepting formatting functions or using locales would be acceptable.
* If we go the hacky way of doing search&replace on the output, then let's do it in realrate-private and not here
* Replacing all dots can easily replace far more than only the decimal points. It should at least check that the dot is part of a number (e.g. a `r'\d[.]\d'` regex).

My suggestion:
Expect that the locale is correctly set before `create_and_save_graph` is called and use https://docs.python.org/3/library/locale.html#locale.format_string for the number formatting in the graph.